packages/quantic/force-app/main/default/lwc/quanticTooltip/quanticTooltip.css

```diff
@@ -1,7 +1,7 @@
 .tooltip__content {
   left: 50%;
   transform: translate(-50%, -100%);
-  top: -0.7rem;
+  top: -0.4rem;
```

Comment:
I noticed in your videos that the tooltip arrow was now aligned with the icon _inside_ the button now instead of being above the entire button.

Is that expected?

Reply:
Yes I found a bit weird that the tooltip is kinda disconnected from the button and completely above  it.